seatunnel-e2e/seatunnel-connector-v2-e2e/connector-elasticsearch-e2e/src/test/java/org/apache/seatunnel/e2e/connector/elasticsearch/ElasticsearchIT.java

```diff
@@ -179,6 +181,17 @@ public void testElasticsearchWithFullType(TestContainer container)
                 esRestClient.getIndexDocsCount("st_index_full_type_target").get(0).getDocsCount());
     }
 
+    @TestTemplate
+    public void testElasticsearchWithoutSchema(TestContainer container)
+            throws IOException, InterruptedException {
+        Container.ExecResult execResult =
+                container.executeJob("/elasticsearch/elasticsearch_source_and_sink.conf");
+        Assertions.assertEquals(0, execResult.getExitCode());
+        List<String> sinkData = readSinkDataWithOutSchema();
+        // for DSL is: {"range":{"c_int":{"gte":10,"lte":20}}}
+        Assertions.assertIterableEquals(mapTestDatasetForDSL(), sinkData);
```

Comment:
I found that other test cases also use the same Sink index `st_index2`, so I think it is necessary to delete the index on the sink side before executing the task to ensure that the data is not contaminated by other test cases.

Reply:
This is for comparing data

Reply:
I mean this test case and https://github.com/apache/seatunnel/pull/6425/files#diff-4ae713cc819c2f12617dfe46ece4360566083e9a54519fce3c470b235766d020R162 both use `st_index2` as index which write data into. Therefore, we need to clear the data in `st_index2` before executing the test case, so as to prove that the data was written by the current use case.